src/tags.ts

```diff
@@ -51,6 +51,21 @@ export interface TagsBase {
    */
   classification: SecurityClassification;
 
+  /**
+   * Criticality of the resources
+   */
+  criticality: 'critical' | 'high' | 'medium' | 'low';
```

Comment:
not used in any of the `tag()`?

Reply:
Do we have a formal definition of what a criticality is? what does it take to be "high" ? 

Is this related to the standard CIA triad where we should have other tags like integrity and availability requirements?

Reply:
@blacha the ownership for defining 'Criticality' is on the squads, as in which service is 'Critical' or 'High'

Reply:
I would probably allow each group(LI/STEP) to define these based on the operational SLAs of respective groups on different stacks.
As long as we can agree the 4 enumerated values are good enough to play with. 
I would put "defining what each rating means for all of LINZ" in a too hard bucket for now.
But in general an example could be "AuthN" stack is critical vs "Common email service" stack could be medium in STEP context.